frontend/components/buttons/Button/_styles.scss

```diff
@@ -245,8 +245,9 @@ $base-class: "button";
         }
       }
 
-      // Copy button uses stroke color instead of fill color to darken
-      &.input-field__copy-value-button {
+      // These buttons use stroke color instead of fill color to darken
+      &.input-field__copy-value-button,
+      &.windows-details-button {
```

Comment:
Random fix where hover was wonky with the stroke/fill color of the chevron

<img width="1230" alt="Screenshot 2024-01-19 at 10 04 52 AM" src="https://github.com/fleetdm/fleet/assets/71795832/79aa3f95-7ed1-4bc0-887a-aaa8bdaf0a11">